src/providers/ogr/qgsgeopackagedataitems.cpp

```diff
@@ -525,9 +525,23 @@ bool QgsGeoPackageAbstractLayerItem::deleteLayer()
   }
   else
   {
-    QMessageBox::information( nullptr, tr( "Delete Layer" ), tr( "Layer <b>%1</b> deleted successfully." ).arg( mName ) );
     if ( mParent )
+    {
+      QgsGeoPackageConnectionItem *connectionParentItem = qobject_cast<QgsGeoPackageConnectionItem *>( mParent );
+      if ( connectionParentItem )
+      {
+        if ( QMessageBox::question( nullptr, QObject::tr( "Delete Layer" ), QObject::tr( "The layer <b>%1</b> deleted successfully."
```

Comment:
My english is bad but maybe better:
```suggestion
        if ( QMessageBox::question( nullptr, QObject::tr( "Delete Layer" ), QObject::tr( "The layer <b>%1</b> was deleted successfully."
```

Reply:
Hm... It's for sure better. But probably "has been deleted"? Not sure.

Reply:
I would also think "has been" deleted if I go by my feeling, but better get confirmation from @nyalldawson or @NathanW2 or @timlinux or someone else who is native

Reply:
https://english.stackexchange.com/a/101351